protocols/seen/overview.md

```diff
@@ -0,0 +1,19 @@
+# Seen.haus
+
+seen.haus is an auction house that brings the physical and digital worlds together. Seen.haus establishes itself as a trustworthy platform for creators and investors offering limited-edition high-quality pieces of digital and physical artworks.
+
+By leveraging blockchain technology, seen.haus provides immutable and publicly verifiable proof of authenticity for its listed artworks that permanently links each physical art piece with its digital representation.
+
+## Additional Links
```

Comment:
I think that this should be a `#` level heading. 
And this overview is missing a few sections such as `Join our DAO` (I see you have their links below), `governance` and potentially `overview`

Reply:
Added and formattted